<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
  background-color: #FAFBFC;
  border: 1px solid rgba(27, 31, 35, 0.15);
  border-radius: 6px;
  color: #24292E;
  font-family:  -apple-system, system-ui, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;;
  font-size: 16px;
  line-height: 18px;
  padding: 3px 16px;
}

QPushButton:hover {
  background-color: #F3F4F6;
}

QPushButton:disabled { background-color: #bdc3c7; color: #7f8c8d; }</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="btnOpen">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>-apple-system</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="title">
        <string>BLANK DOCUMENT PRESETS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProjectDetails">
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>PROJECT DETAILS</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelProjectName">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Project name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editProjectName">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>256</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelWidth">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editWidth">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelHeight">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editHeight">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maxLength">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="labelBackColor">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Background color:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="labelR">
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>R:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editR">
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>255</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="labelG">
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>G:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editG">
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>255</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="labelB">
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>B:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editB">
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>255</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnColor">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="font">
           <font>
            <family>-apple-system</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCreate">
          <property name="font">
           <font>
            <family>-apple-system</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
